package.json

```diff
@@ -13,6 +13,7 @@
     "start:mock:win": "set VITE_USE_MOCK=true&& npm start",
     "start:mock:unix": "export VITE_USE_MOCK=true && npm start",
     "test": "cypress run",
+    "test:vitest": "vitest",
```

Comment:
liten heads up om at vi burde se på hvordan vi kan få kjørt disse automatisk for hver PR, på samme måte som cypress-testene. Tror det passer best som en egen jobb i AWS, og at det blir et eget innslag i `codebuild_validation.yml`. Husker ikke helt hvordan vi satt opp ting der da, så blir fort litt prøving og feiling også. Det kan godt gjøres i en egen PR senere også :)